build.sh

```diff
@@ -56,6 +56,9 @@ JXL=true
 # Support for AVIF, enabled by default
 AVIF=true
 
+# Partial support for SVG via resvg, disabled by default because it's incomplete and relatively rarely useful feature.
+SVG=false
```

Comment:
I think we could enable this by default when building as a dynamic module, allowing users to opt-in by doing:
```js
const vips = await Vips({
  // Enable SVG load support
  dynamicLibraries: ['vips-resvg.wasm']
});
```

Let me know if I should experiment with this. :)

Reply:
Done.